lib/blacklight/configuration.rb

```diff
@@ -166,12 +168,12 @@ def initialize(hash = {})
     end
 
     def document_model
-      super || ::SolrDocument
+      self[:document_model] || ::SolrDocument
```

Comment:
It looks like ostruct 0.3.2 fixes the `super` bug. I don't know if we want to pin the gem to >= 0.3.2 to get the fix?